doc/sample_systemd_script/assembl.service

```diff
@@ -15,8 +15,8 @@ Group=www-data
 
 Type=simple
 Restart=on-failure
-ExecStart=/bin/bash -c '. $ASSEMBL_PATH/venv/bin/activate; exec supervisord -c $ASSEMBL_PATH/supervisord.conf --nodaemon'
-ExecStartPost=/bin/bash -c 'sleep 5; . $ASSEMBL_PATH/venv/bin/activate; supervisorctl -c $ASSEMBL_PATH/supervisord.conf start prod:uwsgi >/tmp/supervisordebug.txt'
+ExecStart=/bin/bash -c 'test -f $ASSEMBL_PATH/local.ini && source $ASSEMBL_PATH/venv/bin/activate && exec supervisord -c $ASSEMBL_PATH/supervisord.conf --nodaemon'
```

Comment:
This file has a template, shouldn't that file be removed before the merge ?

Reply:
Actually I think it should be removed , since all the templates are now in templates/system